gulp/tasks/copy.js

```diff
@@ -1,100 +1,108 @@
   so SVG references in CSS get resolved.
 */
-const iconSrc = `${ paths.modules }/@cfpb/cfpb-icons/src/icons/*.svg`;
+const iconSrc = `${paths.modules}/@cfpb/cfpb-icons/src/icons/*.svg`;
 
 /**
  * Generic copy files flow from source to destination.
  * @param {string} src The path to the source files.
  * @param {string} dest The path to destination.
  * @returns {Object} An output stream from gulp.
  */
-function _genericCopy( src, dest ) {
-  return gulp.src( src )
-    .pipe( gulpChanged( dest ) )
-    .on( 'error', handleErrors )
-    .pipe( gulp.dest( dest ) );
+function _genericCopy(src, dest) {
+  return gulp
+    .src(src)
+    .pipe(gulpChanged(dest))
+    .on("error", handleErrors)
+    .pipe(gulp.dest(dest));
 }
 
-gulp.task( 'copy:root', () => {
+gulp.task("copy:root", () => {
   const stream = _genericCopy(
-    `${ paths.unprocessed }/root/**/*`,
+    `${paths.unprocessed}/root/**/*`,
     paths.processed
   );
   return stream;
-} );
+});
 
-gulp.task( 'copy:icons:main', () => {
-  const stream = _genericCopy(
-    iconSrc,
-    `${ paths.processed }/icons/`
-  );
+gulp.task("copy:icons:main", () => {
+  const stream = _genericCopy(iconSrc, `${paths.processed}/icons/`);
   return stream;
-} );
+});
 
-gulp.task( 'copy:icons:oah', () => {
+gulp.task("copy:icons:oah", () => {
   const stream = _genericCopy(
     iconSrc,
-    `${ paths.processed }/apps/owning-a-home/icons/`
+    `${paths.processed}/apps/owning-a-home/icons/`
   );
   return stream;
-} );
+});
 
-gulp.task( 'copy:icons:r3k', () => {
+gulp.task("copy:icons:r3k", () => {
   const stream = _genericCopy(
     iconSrc,
-    `${ paths.processed }/apps/regulations3k/icons/`
+    `${paths.processed}/apps/regulations3k/icons/`
   );
   return stream;
-} );
+});
 
-gulp.task( 'copy:icons:prepaid', () => {
+gulp.task("copy:icons:prepaid", () => {
   const stream = _genericCopy(
     iconSrc,
-    `${ paths.processed }/apps/prepaid_agreements/icons/`
+    `${paths.processed}/apps/prepaid_agreements/icons/`
   );
   return stream;
-} );
+});
 
-gulp.task( 'copy:lightbox2', () => {
+gulp.task("copy:lightbox2", () => {
   const stream = _genericCopy(
-    `${ paths.modules }/lightbox2/dist/**/*`,
-    `${ paths.processed }/lightbox2`
+    `${paths.modules}/lightbox2/dist/**/*`,
+    `${paths.processed}/lightbox2`
   );
   return stream;
-} );
-
+});
 
-gulp.task( 'copy:icons',
+gulp.task(
+  "copy:icons",
   gulp.parallel(
-    'copy:icons:main',
-    'copy:icons:oah',
-    'copy:icons:r3k',
-    'copy:icons:prepaid'
+    "copy:icons:main",
+    "copy:icons:oah",
+    "copy:icons:r3k",
+    "copy:icons:prepaid"
   )
 );
 
-gulp.task( 'copy:moneytools', () => {
-  const app = 'mmt-my-money-calendar';
+gulp.task("copy:mymoneycalendar", () => {
+  const app = "mmt-my-money-calendar";
+
+  return gulp
+    .src(path.join(paths.processed, `apps/${app}/js/service-worker.js`))
+    .pipe(gulp.dest(`./cfgov/jinja2/v1/${app}/`));
+});
+
+gulp.task("copy:costofcredittool", () => {
```

Comment:
Like the other service worker related feedback, this gulp task only needs to exist if the app is going to need a service worker.